### ISSUE TITLE
Multiple instances

### ISSUE THREAD

**Issue by ShaneMcC** [csmith/Dotege#1]
Sometimes it would be nice to be able to run multiple instances, each generating different templates and matching only a subset of containers. At the moment they would both end up getting certs and generating templates for all of the same containers.

(Eg I essentially want a single-purpose instance that just deals with one container and proxies it on port 8080 or so, but I might want another instance elsewhere on the same box that deals with normal stuff on 80/443)

I think there are 2 ways of doing this, either would work:

1. Ability to change the label names (or change the prefix before `<auth|proxy|vhost>` so `com.chameth` would be the default but I could specify `dotege.main` or `dotege.8080` for example)

2. Add an extra label something like `com.chameth.dotege.tag` that can specify 1 or more tags on a container, and then a `DOTEGE_TAGS` env var to specify which tags to match. (`OR` matching I think makes the most sense?)

**Comment by ShaneMcC** [comment]
Either approach lets me do what I want, I think #2 may be more flexible? :man_shrugging:

**Comment by csmith** [comment]
Yeah, at one point the labels were half configurable and it got messy quickly - #2 is probably the way to go.
